project/Dependencies.scala

```diff
@@ -86,7 +87,9 @@ object Dependencies {
 
   // Sub-project dependencies, added in addition to any dependencies inherited from .dependsOn().
 
-  val gcsFileSystemDependencies = baseDependencies ++ googleApiClientDependencies ++ googleCloudDependencies
+  val gcsFileSystemDependencies = baseDependencies ++ googleApiClientDependencies ++ googleCloudDependencies ++ List (
+    "com.github.pathikrit" %% "better-files" % betterFilesV
```

Comment:
Don't duplicate dependencies. If you're adding better-files to gcs, since engine inherits from gcs, then remove them from the engine.